code/modules/projectiles/targeting/targeting_overlay.dm

```diff
@@ -174,6 +176,10 @@ obj/aiming_overlay/proc/update_aiming_deferred()
 	moved_event.register(owner, src, /obj/aiming_overlay/proc/update_aiming)
 	moved_event.register(aiming_at, src, /obj/aiming_overlay/proc/target_moved)
 	destroyed_event.register(aiming_at, src, /obj/aiming_overlay/proc/cancel_aiming)
+var/aimcooldown //this is hacky im sorry.
```

Comment:
Not sure that'll work, as it's used in 
`obj/aiming_overlay/proc/aim_at(var/mob/target, var/obj/thing)
	if (aimcooldown > world.time) ` As well.